.github/workflows/test.yml

```diff
@@ -0,0 +1,54 @@
+name: Lint and Test code
+
+on:
+  pull_request:
+  schedule:
+    # run once a week on early monday mornings
```

Comment:
Is this in 02:22 UTC?

Reply:
Great question! we'll find out on Sunday :)